### PR TITLE
Audit of test/* OWNERS files

### PR DESCRIPTION
**Pull request by timothysc**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
I did an audit of the OWNERS files under test/* and updated according to looking at rough github data over the last ~6-ish-months.  No `approvers` have been removed, they've only been shuffled to emeritus 

**Which issue(s) this PR fixes**:

xref: https://github.com/kubernetes/community/issues/1808
xref:  https://github.com/kubernetes/community/issues/4149

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```
NONE
```
/assign @dims @mrbobbytables

### CODE REVIEW

**Comment by dims** [comment]
@xichengliudui @stevesloka @rramkumar1 @ncdc @shyamjvs @vishh @mml @d-nishi @dchen1107 @fgrzadkowski @piosz @balajismaniam @luxas @ixdy @piosz @barney-s @crimsonfaith91 @gmarek @rmmh Please take a look!

**Comment by ncdc** [comment]
My move LGTM

**Comment by timothysc** [comment]
@dims we should timeout, at a couple of weeks b/c it was clear from some folks github history that they have moved on.

**Comment by rramkumar1** [comment]
LGTM for my change

**Comment by mrbobbytables** [comment]
Agreed, we should hold for timebox on this.  Undoing the change is quick and easy enough, so not overly concerned. What do you think is reasonable? 2 weeks? 3?

I'll do a follow-up PR to `sigs.yaml` to add the OWNERS file references to the appropriate sigs.

/hold
(for timebox)

File: test/typecheck/OWNERS
Comment (BenTheElder) [inline test/typecheck/OWNERS]: cc @cblecker -- I'd like to swap in on this one rather than have n=1, I think I already have test/ via an alias and I'm already in reviewers / have sent a PR or two to this.

Comment (BenTheElder) [reply test/typecheck/OWNERS]: there's also a related directory in third_party/ that should be updated (though understandable if it's another PR)
https://github.com/kubernetes/kubernetes/blob/master/third_party/go-srcimporter/OWNERS

**Comment by BenTheElder** [comment]
thanks for doing this. 

suggestion: it might also be a good idea to audit users in OWNERS_ALIASES aliases entries that transitively are reviewers / approvers here (can certainly be another PR rather than complicating this one, but wanted to mention...)

**Comment by mrbobbytables** [comment]
> suggestion: it might also be a good idea to audit users in OWNERS_ALIASES aliases entries that transitively are reviewers / approvers here (can certainly be another PR rather than complicating this one, but wanted to mention...)

@BenTheElder I'll be sending a PR and tagging groups about it next week. 👍

**Comment by stevesloka** [comment]
LGTM for my change

**Comment by timothysc** [comment]
@BenTheElder 

>  it might also be a good idea to audit users in OWNERS_ALIASES aliases entries that transitively are reviewers / approvers here (can certainly be another PR rather than complicating this one, but wanted to mention... 

Yes, my plan is to do this in chunks till I have enough data, then update aliases for *, but "baby-steps".

**Comment by xichengliudui** [comment]
I'm sorry about that, because most of my energy is focused on kubernetes/website repo.

**Comment by timothysc** [comment]
Hmm I can no longer skip tests, this seems wasteful :-( 

/test pull-kubernetes-e2e-gce-100-performance
/test pull-kubernetes-kubemark-e2e-gce-big
/test pull-kubernetes-conformance-kind-ipv6
/test pull-kubernetes-conformance-image-test

**Comment by BenTheElder** [comment]
Non blocking tests shouldn't block merge and should be skippable.

The conformance image jobs are all rather flaky, something to do with how the image is run.

The scale jobs probably hit https://github.com/kubernetes/kubernetes/issues/83493#issuecomment-539306463

File: test/typecheck/OWNERS
Comment (cblecker) [reply test/typecheck/OWNERS]: I'm okay with adding @BenTheElder , but I'm also okay with it being done in a follow up.

**Review by cblecker — APPROVED** [review]
/lgtm
/approve

**Comment by cblecker** [comment]
As for moving on removing inactive folks, if they haven't been active in that long, I'm fine with moving quickly if there is consensus among other owners. If the person wants to come back, they are only a PR away from doing that.

Suggestion: 72 hour lazy consensus of `test/` owners. WDYT, @timothysc ?

**Comment by timothysc** [comment]
> Suggestion: 72 hour lazy consensus of test/ owners. 

sgtm

**Comment by timothysc** [comment]
rebased + timeout 
/hold cancel 
/skip 
@cblecker ^ for lgtm

File: test/typecheck/OWNERS
Comment (BenTheElder) [reply test/typecheck/OWNERS]: filed https://github.com/kubernetes/kubernetes/pull/84020 to follow up :smile:

**Pull request merged**
